<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>消防设备电源监控系统-集成平台</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
color: rgb(255, 255,255);
background-color: rgb(60,63,65);

}
QToolButton{
font: 22pt &quot;楷体&quot;;
border:2px none;
color: rgb(85, 170, 127);
border-color: rgb(60,63,65);
}

QToolButton:hover{
font: 24pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
background-color: rgb(60,63,65);
}

</string>
   </property>
   <widget class="QToolButton" name="tBtnConfHostInfo">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>220</y>
      <width>150</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string>网络设置</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/polaroids.png</normaloff>:/Image/polaroids.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnDisplayInfo">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>55</y>
      <width>150</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string>监控信息</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/run.png</normaloff>:/Image/run.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnRecordInfo">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>385</y>
      <width>150</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string>记录查询</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/volume.png</normaloff>:/Image/volume.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>55</y>
      <width>1130</width>
      <height>660</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 41);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnAboutInfo">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>550</y>
      <width>150</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string>软件信息</string>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/Image/updata.png</normaloff>:/Image/updata.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60,63,65);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>55</y>
      <width>150</width>
      <height>666</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60,63,65);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>260</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>04b_21</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 232, 56);
border:none;
</string>
    </property>
    <property name="digitCount">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>731</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LC-ACS SER 消防设备电源监控系统-集成平台</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/Image/LOGO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>tBtnRecordInfo</zorder>
   <zorder>tBtnDisplayInfo</zorder>
   <zorder>tBtnConfHostInfo</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>tBtnAboutInfo</zorder>
   <zorder>label</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
